openshift/cirras-underwriting-ui-configmap.yaml

```diff
@@ -14,6 +14,9 @@ metadata:
       fieldsType: FieldsV1
 immutable: false
 data:
+  TOMCAT_PORT: '#{TOMCAT_PORT}#'
+  TOMCAT_MAX_THREADS: '#{TOMCAT_MAX_THREADS}#'
+  CONNECTION_TIMEOUT: '#{CONNECTION_TIMEOUT}#'
```

Comment:
There is no CONNECTION_TIMEOUT in openshift-deploy?

Reply:
Added missing property to openshift-deploy.yml